Sources/RTVIClientIOS/RTVIClientDelegate.swift

```diff
@@ -73,18 +73,22 @@ public protocol VoiceClientDelegate: AnyObject {
 
     /// Invoked when bot transcript data is avaiable.
     func onBotTranscript( data: String)
+    
+    /// Invoked when received the bot transcription from the LLM.
+    func onBotText(data: BotLLMText)
 
     /// Invoked when we receive an error message from the backend
     func onError( message: String)
 
     /// Invoked when a message from the backend is received which was not handled by the VoiceClient or a registered helper.
-    func onGenericMessage (message:VoiceMessageInbound)
+    func onGenericMessage (message:RTVIMessageInbound)
     
     /// Invoked when the set of available cam/mic tracks changes.
     func onTracksUpdated(tracks: Tracks)
 }
 
-public extension VoiceClientDelegate {
+public extension RTVIClientDelegate {
+    func onBotText(data: BotLLMText) {}
```

Comment:
I've called this `onBotLLMText` on Android, there is also `onBotTTSText`.

Reply:
I was following the way that our javascript API it is implemented. But I guess it makes more sense the way it is on Android.
I will change that to mnake the Android behavior.